.github/workflows/release_ios_staging.yml

```
@@ -0,0 +1,114 @@
+name: Release iOS Staging App to Firebase
+
+on:
+  push:
+    branches: [develop, chore/#5-set-up-release-firebase-staging-workflow]
+
+
+jobs:
+  shared_test:
+      name: Running tests in Shared module
+      runs-on: ubuntu-latest
+      defaults:
+        run:
+          working-directory: shared
+      steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Java JDK
+        uses: actions/setup-java@v2.1.0
+        with:
+          distribution: 'adopt'
+          java-version: '11'
+
+      - name: Running tests
+        run: |
+          cd ..
+          ./gradlew clean shared:test
+
+  ios_release_staging:
+    name: Automatic Release iOS Staging App to Firebase
+    runs-on: macos-latest
+    defaults:
+        run:
+          working-directory: iosApp
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Set up JAVA 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install SSH key - For accessing match repo
+        uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+      - uses: actions/cache@v3
+        id: bunlderCache
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pods-
+
+      - name: Bundle install
+        run: bundle install --path vendor/bundle
+
+      - name: Paste Google-Info.plist from .secrets
+        env:
+          IOS_GOOGLE_SERVICE_INFO_STAGING: ${{ secrets.IOS_STAGING_GOOGLE_INFO_PLIST }}
+        run: |
+          mkdir -p PhongKMMIC/Configurations/Plists/GoogleService/Staging
+          touch PhongKMMIC/Configurations/Plists/GoogleService/Staging/GoogleService-Info.plist
+          echo $IOS_GOOGLE_SERVICE_INFO_STAGING | base64 --decode > ./PhongKMMIC/Configurations/Plists/GoogleService/Staging/GoogleService-Info.plist
```

I think we can improve this without touch (please validate this)
```suggestion
          echo $IOS_GOOGLE_SERVICE_INFO_STAGING | base64 --decode >> ./PhongKMMIC/Configurations/Plists/GoogleService/Staging/GoogleService-Info.plist
```